content/v2.0/actions/control-flow.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Control Flow
+order: 110
+aliases:
+  - "/actions/control-flow"
+---
+
+## Callbacks
+
+If we want to execute some logic before and/or after `#handle` is executed, we can use a callback.
+Callbacks are useful to declutter code for common tasks like checking if a user is signed in, set a record, handle 404 responses or tidy up the response.
```

Comment:
What do you mean by `set a record`?

Reply:
I see it like: fetching a record to use in order to process/check authorize access to. This is not changed from guides 1.3.

Reply:
I have removed this mention for clarity. It's not necessary to show the point.